rero_ils/modules/local_fields/jsonschemas/local_fields/local_field-v0.0.1.json

```diff
@@ -0,0 +1,166 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "title": "Local fields",
+  "description": "Local fields",
+  "additionalProperties": false,
+  "required": [
+    "$schema",
+    "pid",
+    "fields"
+  ],
+  "propertiesOrder": [
+    "fields"
+  ],
+  "properties": {
+    "$schema": {
+      "title": "Schema",
+      "description": "Schema to validate local_field records against.",
+      "type": "string",
+      "minLength": 9,
+      "default": "https://ils.rero.ch/schema/local_fields/local_field-v0.0.1.json"
+    },
+    "pid": {
+      "title": "Local field ID",
+      "type": "string",
+      "minLength": 1
+    },
+    "organisation": {
+      "title": "Organisation",
+      "type": "object",
+      "properties": {
+        "$ref": {
+          "title": "Organisation",
+          "type": "string",
+          "pattern": "^https://ils.rero.ch/api/organisations/.*?$"
+        }
+      }
+    },
+    "parent_ref": {
```

Comment:
I think `parent` only is enough.  do not need to the `_ref`

Reply:
same than @BadrAly

Reply:
done